src/components/Header/Header.module.css

```diff
@@ -1,21 +1,5 @@
 .headerToolbar {
```

Comment:
As far as the grey vertical bar/border it only needs to be present starting at screen width of 1280px then you align-items: flex-end on the headerToolbar.
![Screenshot 2023-09-11 at 2 32 27 PM](https://github.com/gogonzogo/Final-Team-Project/assets/114868158/5e7c5496-4806-4066-9d3d-dbec70b4d2a6)